### ISSUE TITLE
EMI Compatibility

### ISSUE THREAD

**Issue by littlelakehouse** [mezz/ItemZoom#42]
as the title says

**Comment by xAlicatt** [comment]
Was coming to suggest this as well. +1

**Comment by hgjensen** [comment]
+1

**Comment by TroubleDarkness** [comment]
+1, extremely needed feature.

**Comment by 7OOTnegaTerces** [comment]
I also agree, this is really needed!  I would like to also request that this fix be included in the 1.20.1 version, the version I am currently using (and will be for the time being).  Thanks!!!

**Comment by SlayDigger** [comment]
+1

**Comment by 7OOTnegaTerces** [comment]
I rescind my request, the Item Zoom I originally got is now working with EMI, so... good job?!? ;P

**Comment by xAlicatt** [comment]
> I rescind my request, the Item Zoom I originally got is now working with EMI, so... good job?!? ;P

Which version of the mod/EMI/minecraft and what mod loader and version?

**Comment by 7OOTnegaTerces** [comment]
> > I rescind my request, the Item Zoom I originally got is now working with EMI, so... good job?!? ;P
> 
> Which version of the mod/EMI/minecraft and what mod loader and version?

emi-1.1.10+1.20.1+forge
itemzoom-1.20.1-2.9.0
Minecraft 1.20.1
NeoForge (final version for 1.20.1)

I've got a massive mod list I put together, and I suspect that one of the other mods I removed was causing the interference, but good luck finding out which one...

**Comment by chjs0711** [comment]
Does work with items in inventory but not with EMI.
These are literally the only mods I am using EMI latest version (1.1.13) + Item Zoom latest version (2.9.0) + forge 47.3.7 1.20.1

So yeah EMI compatibility sounds good!

**Comment by 7OOTnegaTerces** [comment]
I take it back, item zoom isn't working with EMI.  It wasn't working at all for a while (hence my confused posts above), but now it only works with inventory items, but not anything listed in EMI....

**Comment by sherpascott** [comment]
I would love this feature too!

**Comment by JohnGoten** [comment]
Can confirm using on 1.21.1 with
NeoForge 21.1.115
ItemZoom 3.0.0
EMI 1.1.19

**Comment by 7OOTnegaTerces** [comment]
> Can confirm using on 1.21.1 with NeoForge 21.1.115 ItemZoom 3.0.0 EMI 1.1.19

Can confirm what? That it’s not working or that it is?!?

**Comment by JohnGoten** [comment]
> Can confirm what? That it’s not working or that it is?!?

_sigh_

I am pretty sure it was obvious that the issue in question is still present using either of the latest on NF 1.21.1
No clue why you'd be asking so passive aggressive rather than think even for a minute, jfc. No need to start unnecessary heat on something that was pretty clear to understand

**Comment by 7OOTnegaTerces** [comment]
> > Can confirm what? That it’s not working or that it is?!?
> 
> _sigh_
> 
> I am pretty sure it was obvious that the issue in question is still present using either of the latest on NF 1.21.1 No clue why you'd be asking so passive aggressive rather than think even for a minute, jfc. No need to start unnecessary heat on something that was pretty clear to understand

I’m sorry that I came across as passive aggressive, that wasn’t my intention, rather I was hoping, in vain apparently, that you might have been letting all of us know that the problem had been resolved in the latest version. 😓  Yes, that was the less likely scenario , but when do hopes and dreams care about pesky things like logic?!? 😳  I probably should have included some emojis to better convey my feelings.  That’s the problem with text based communication, it lacks the tone, body language, and other communication cues that can help convey extra context.  I should have realized that and included my own context, but I was in a hurry and didn’t, I’m sorry for that oversight.

I’m also not helped by the fact that there isn’t such a thing as a universal human experience or universal common sense.  Just the common sense that is shared by our close social circle.  What’s obvious and natural to one human isn’t to another. That’s why you can give two humans the exact same set of data, and yet they can come to wildly different conclusions. And while this is helpful when it comes to problem solving, it can be an unexpected trap to communication, even when it should be expected!  For example, just because I like to use exclamation points to express energy/excitement, and just because that is a valid use of them, that doesn’t mean that my readers will interpret them that way! 🤷🏽‍♂️🤦🏽‍♂️

So thank you for letting us, and more importantly the mod’s developers, know that this is still an issue!  I hope my hasty response didn’t spoil too much of your day (triple exclamation points because I really, REALLY, mean that)!!! 🥹  And the next time you read a post from a dunce like me, just remind yourself that “common sense” is really actually a myth and forgive them for their communication blunders.

Wishing you the best of all possible days, and an amazing Minecraft experience!!!😁🎉

**Comment by BitBork** [comment]
Great mod!
Would love this feature as well, I'm on Forge 1.20.1 which it doesn't seem to work on
Forge 47.4.0 - 1.20.1 + EMI 1.1.22 - 1.20.1 + Itemzoom 2.9.0

**Comment by djrcapo** [comment]
Hey! I made a fork for this mod with compatibility with EMI. Check them and feel free to tell if it works!

- [ItemZoom v2.9.0 - 1.20.1](https://github.com/djrcapo/ItemZoom-EMI-Compat/releases/tag/v2.9.1-emi)
- [ItemZoom v3.0.0 - 1.21](https://github.com/djrcapo/ItemZoom-EMI-Compat/releases/tag/v3.0.1-emi)

**Comment by ReeCodes** [comment]
> Hey! I made a fork for this mod with compatibility with EMI. Check them and feel free to tell if it works!
> 
> * [ItemZoom v2.9.0 - 1.20.1](https://github.com/djrcapo/ItemZoom-EMI-Compat/releases/tag/v2.9.1-emi)
> * [ItemZoom v3.0.0 - 1.21](https://github.com/djrcapo/ItemZoom-EMI-Compat/releases/tag/v3.0.1-emi)

Will this ever become an official mod you can get through curseforge?
